src/util/defaults.ml

```diff
@@ -116,6 +117,7 @@ let _ = ()
       ; reg Analyses "ana.osek.flags"      "[]"    "List of global variables that are flags."
       ; reg Analyses "ana.osek.def_header" "true"  "Generate TASK/ISR macros with default structure"
       ; reg Analyses "ana.int.wrap_on_signed_overflow" "false" "Whether to assume wrap-around behavior on signed overflow. If set to true, assumes two's complement representation of signed integers. If set to false, goes to top on signed overflow."
+      ; reg Analyses "ana.int.no_signed_overflow" "true" "Assume there will be no signed overflow."
```

Comment:
Why is this the default?

Reply:
I am not suggesting that we should not have this for octagons. However, this is decidedly not what all the other domains (e.g. intervals) do, where we very pain-stakingely made sure to go to top in such cases (which imo is the philosophically sounder thing (whatever sound means for programs with UB))

In general for all of our domains, I am against making the assumption of no signed overflows by default.

For the octagons, one might want this behavior though. As currently this is only used by octagons, I would suggest renaming the option to reflect that.

Reply:
> For the octagons, one might want this behavior though. As currently this is only used by octagons, I would suggest renaming the option to reflect that.

I'll rename this option (and other octapron options) into an option group like `ana.octapron`.